capsules/src/signbus/SignbusLayers.md

```diff
@@ -0,0 +1,87 @@
+# Signbus Communication Layers
+
+
+-\ USERLAND <br />
+&nbsp;&nbsp;  -\ app_layer <br />
+&nbsp;&nbsp;&nbsp;&nbsp;    -\ protocol_layer <br />
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;    -\ io_layer <br />
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;        -\ port_layer <br />
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;        -\ I2C DRIVER <br />
+
+
+## app_layer
+Userland buffers and callbacks.
```

Comment:
This layer doesn't actually interact with userland (in practice if we wanted to connect this protocol to userland, there would be another layer on top of `app_layer` that would do so). Instead this handles "Application messages and callbacks", with anything above this layer being application-specific.